.editorconfig

```diff
@@ -13,6 +13,6 @@ trim_trailing_whitespace = true
 [*.bat]
 end_of_line = crlf
 
-[*.yml]
+[*.{yml,xml,json,ctp,css}]
```

Comment:
Hehe, there's where the mess and discussion starts once you abandon proper tab indentation :P